firmware/config/boards/hellen/cypress/rusefi_hw_enums.h

```diff
@@ -234,7 +232,10 @@ typedef enum __attribute__ ((__packed__)) {
 } brain_pin_e;
 
 /* Plase keep updating this define */
-#define BRAIN_PIN_LAST_ONCHIP	GPIOK_15
+#define BRAIN_PIN_ONCHIP_LAST	GPIOK_15
+#define BRAIN_PIN_ONCHIP_PINS	(BRAIN_PIN_ONCHIP_LAST - GPIOA_0 + 1)
+#define BRAIN_PIN_LAST 			DRV8860_PIN_16
+#define BRAIN_PIN_TOTAL_PINS	(BRAIN_PIN_LAST - GPIOA_0 + 1)
```

Comment:
can these be values within the enum? remember multiple elements can share the same value if necessary.

Reply:
Yes, but no.
AFAIR this header is parsed and used to generate auto_generated_enums.cpp and switch case inside. So we need to modify parser to support enum elements with same value...